programs/programs/tx/pe/member.py

```
@@ -122,6 +122,43 @@ def member_value(self, member: HouseholdMember):
+    Notes:
+    - The citizenship eligibility is handled at the program configuration level
+      (legal_status_required), not in this calculator.
+    - We do not ask users whether they have an emergency medical condition in the screener.
+      Instead, this requirement is communicated in the program's description so users understand
+      they must have a qualifying condition to receive benefits.
+    """
+
+    pe_inputs = [
+        *Medicaid.pe_inputs,
+        dependency.household.TxStateCodeDependency,
+    ]
```

Ruff (0.14.10)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)